src/cmds/scripts/pbs_init.d.in

```diff
@@ -668,105 +626,11 @@ find_bgl_environment ()
 
 }
 
-# Wrapper for SGI's chkfeature utility, q.v..  "has_chkfeature" is initialized
-# in the site_mom_startup function.
-sgi_chkfeature()
-{
-	if [ $has_chkfeature -ne 0 ]
-	then
-		chkfeature ${1+"$@"}
-		return $?
-	else
-		return 1
-	fi
-}
-
-# called when we are starting on a SGI ICE system
-# /etc/sgi-compute-node-release exists
-# Make a vnodedef file with cpuset info
-sgi_ice_startup()
-{
-	vnodedefsdir=`dirname $vnodedefs`
-	[ -d $vnodedefsdir ] || mkdir -p -m 0700 $vnodedefsdir
-
-	sh $momscriptdir/sgiICEvnode.sh "$1"  > ${vnodedefs}.new
-
-	if [ ! -f $vnodedefs ] || ! cmp -s $vnodedefs ${vnodedefs}.new
-	then
-		/bin/cp ${vnodedefs}.new $vnodedefs
-	fi
-	/bin/rm -f ${vnodedefs}.new
-}
-
 site_mom_startup()
 {
-    vnodedefs="${PBS_MOM_HOME}/mom_priv/config.d/PBSvnodedefs"
-    momscriptdir="${pbslibdir}/init.d"
-    SGItopology="/proc/sgi_sn/sn_topology";
-    UVSGItopology="/proc/sgi_uv/topology";
-    SGIcomputenode="/etc/sgi-compute-node-release"
-    if type chkfeature > /dev/null 2>&1
-    then
-	has_chkfeature=1
-    else
-	has_chkfeature=0
-    fi
-    #	If this site is using a pbs_mom that supports CPU sets ...
-    if using_cpuset_mom
-    then
-	# and if there's an SGI topology file (Altix),
-	# generate a placement and vnode definitions file.
-	if  [ -f $SGItopology -o -f $UVSGItopology ]
-	then
-		vnodedefsdir=`dirname $vnodedefs`
-		[ -d $vnodedefsdir ] || mkdir -p -m 0700 $vnodedefsdir
-		awk -f "$momscriptdir/sgigenvnodelist.awk" > $vnodedefs
-	elif [ -f $SGIcomputenode ]
-	then
-		sgi_ice_startup cpuset
-	fi
-    fi
     find_bgl_environment
```

Comment:
We no longer support Blue Gene. I think this function and references to it should be removed.

Reply:
There's a separate ticket to remove Blue Gene.  I think it is best to keep the removal simpler by keeping them separate.